src/main/kotlin/com/launchdarkly/intellij/action/RunCoderefsAction.kt

```diff
@@ -1,87 +0,0 @@
-package com.launchdarkly.intellij.action
```

Comment:
I deleted this file, its not used anywhere, we can always revive it in history if we need it.

Reply:
👀  does this mean we aren't _actually_ using the **Enable Code Refs** plugin settings? 😬

Reply:
AHH ok gotcha